### PR TITLE
Implement UpdateConfigRaw in hyperkit driver

### PR DESCRIPTION
**Pull request by cfergeau**
This will make it possible to update memory size/cpu count of the VM.

### CODE REVIEW

**Comment by praveenkumar** [comment]
Run `go mod tidy` should be part of https://github.com/code-ready/machine-driver-hyperkit/pull/19/commits/d3a63490379810344716f8b2090abc308b6f6e55 one where we are updating the machine version to go.mod instead of a separate commit message.

**Comment by cfergeau** [comment]
> Run `go mod tidy` should be part of [d3a6349](https://github.com/code-ready/machine-driver-hyperkit/commit/d3a63490379810344716f8b2090abc308b6f6e55) one where we are updating the machine version to go.mod instead of a separate commit message.

Running `git mod tidy` on current master (before this patch series) generates a similar diff to the dedicated commit in this PR, so the go mod tidy change is not related to the other commits in this PR. I'll squash it as it's not so important

File: pkg/hyperkit/driver.go [hunk 1]
```diff
@@ -404,3 +409,14 @@ func (d *Driver) getPid() int {
 
 	return config.Pid
 }
+func (d *Driver) UpdateConfigRaw(rawConfig []byte) error {
```
Comment (praveenkumar) [inline pkg/hyperkit/driver.go:412]: This shouldn't be part of https://github.com/code-ready/machine-driver-hyperkit/pull/19/commits/482ba186d3323b5130f26e1b76829063bdce7294 commit.

**Comment by praveenkumar** [comment]
https://github.com/code-ready/machine-driver-hyperkit/pull/19/commits/d1eea888da3be4a6567acdbf489ed0e25fd8f7aa logs says `Set driver version to 0.12.9` but it should be `0.12.8`

File: pkg/hyperkit/constants.go [hunk 2]
```diff
@@ -2,7 +2,7 @@ package hyperkit
 
 const (
 	DriverName    = "hyperkit"
-	DriverVersion = "0.12.7"
+	DriverVersion = "0.12.8"
```
Comment (praveenkumar) [inline pkg/hyperkit/constants.go:5]: :+1:

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**